addons/event_sale/wizard/event_configurator_views.xml

```diff
@@ -5,7 +5,15 @@
         <field name="model">event.event.configurator</field>
         <field name="arch" type="xml">
             <form js_class="event_configurator_form">
-                <group>
+                <field name="has_available_tickets" invisible="1"/>
+                <div invisible="has_available_tickets">
+                    Please note that at least one ticket type must be set for an event. <br/>
```

Comment:
Message is a bit confusing.

Maybe "We could not find a matching event ticket for this product" ?